### PR TITLE
Fixed visibility of Sanger ordered badge in light mode

### LINKED ISSUE #3447: Sanger request not visible
When requesting a Sanger validation the text indicating that you´ve done that is white and not visible in light mode, you can only see the text in dark mode.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3447 

Main branch (white on white):

image

This branch:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Fixed also the default default gene panel badge on case page, from this:

image


To this:

image

**Review by dnil — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ##[]
 ### Fixed
 - Some of the DataTables tables got a bit dark in dark mode
--
+- Visibility of Sanger odered badge on case page, light mode
```
Comment (dnil) [inline CHANGELOG.md:10]:
```suggestion
- Visibility of Sanger ordered badge on case page, light mode
```

**Comment by dnil** [comment]
I found another `bg-default` on the gene_panel page; if you haven't merged already perhaps you could fix it as well? 😸

**Comment by northwestwitch** [comment]
> I found another `bg-default` on the gene_panel page; if you haven't merged already perhaps you could fix it as well?

It the one I've just fixed, the one on my previous comment I think

**Comment by dnil** [comment]
> > I found another `bg-default` on the gene_panel page; if you haven't merged already perhaps you could fix it as well?
> 
> It the one I've just fixed, the one on my previous comment I think

Ah, super yes - I misread the file name!

**Pull request merged**